src/main/scala/org/broadinstitute/dsde/firecloud/service/FireCloudDirectives.scala

```diff
@@ -18,10 +20,25 @@ object FireCloudDirectiveUtils {
     }
     toUri(path).toString
   }
+
+  // TODO: should this pass through any other headers, such as Cookie?
+  /* This list controls which headers from the original request are passed through to the
+   * target service. It is important that some headers are NOT passed through:
+   *     - if the Host header is passed through, it will not match the target service's host,
+   *        and some servers - notably App Engine - will reject the request.
+   *     - if headers that control the http protocol such as Accept-Encoding or Connection
+   *        are passed through, they may not reflect reality. The original request may have
+   *        come from a browser that supports different encodings or connection protocols
+   *        than the service-to-service request we're about to make.
+   *     - if headers that inform the target such as User-Agent, Referer or X-Forwarded-For
+   *        are passed through, they will be misleading, as they reflect the original request
+   *        and not the service-to-service request we're about to make.
+   */
+  final val allowedPassthroughHeaders = List(Authorization, `Content-Type`).map(_.lowercaseName)
+
```

Comment:
I believe this is a functional change from previous passthroughs - we did not send the content-type header. I think this is a worthwhile add, but am totally open to argument we shouldn't do it.